### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by aswin-syras**

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CommentsController < ApplicationController
+ def create
+    @post = Post.find(params[:post_id])
```
Comment (eshaiju) [inline app/controllers/comments_controller.rb:3]: alignment issue

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    @comment = @post.comments.create(comment_params)
+    redirect_to post_path(@post)
+  end
+ 
+  private
+    def comment_params
```
Comment (eshaiju) [inline app/controllers/comments_controller.rb:9]: alignment issue

File: app/views/posts/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<%= form_with scope: :post,url: posts_path, local: true do |form| %>
+   <% if @post.errors.any? %>
```
Comment (eshaiju) [inline app/views/posts/new.html.erb:2]: alignment issue

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (aswin-syras) [reply app/controllers/comments_controller.rb:3]: This is Rubocop auto corrected file

Comment (eshaiju) [reply app/controllers/comments_controller.rb:3]: need 2 space before def I think. Can you run rubocop auto correct once more?

**Pull request merged**